diff --git a/node_modules/@tarojs/webpack5-prebundle/dist/utils/index.js b/node_modules/@tarojs/webpack5-prebundle/dist/utils/index.js
index 9d19f37..4c4f80d 100644
--- a/node_modules/@tarojs/webpack5-prebundle/dist/utils/index.js
+++ b/node_modules/@tarojs/webpack5-prebundle/dist/utils/index.js
@@ -39,7 +39,7 @@ function createResolve(appPath, resolveOptions) {
         cache: true,
         mainFiles: ['index'],
         exportsFields: ['exports'],
-        roots: appPath
+        roots: [appPath]
     };
     const resolver = enhanced_resolve_1.default.create(Object.assign(Object.assign({}, defaultResolveOptions), resolveOptions));
     resolve = function (importer, request) {
